Flow.Launcher.Plugin/SharedCommands/FilesFolders.cs

```diff
@@ -241,5 +237,33 @@ public static string ReturnPreviousDirectoryIfIncompleteString(string path)
 
             return path;
         }
+
+        /// <summary>
+        /// Returns if <paramref name="parentPath"/> contains <paramref name="subPath"/>.
+        /// From https://stackoverflow.com/a/66877016
+        /// </summary>
+        /// <param name="parentPath">Parent path</param>
+        /// <param name="subPath">Sub path</param>
+        /// <param name="allowEqual">If <see langword="true"/>, when <paramref name="parentPath"/> and <paramref name="subPath"/> are equal, returns <see langword="true"/></param>
+        /// <returns></returns>
+        public static bool PathContains(string parentPath, string subPath, bool allowEqual = false)
+        {
+            var rel = Path.GetRelativePath(parentPath.EnsureTrailingSlash(), subPath);
+            return (rel != "." || allowEqual)
+                   && rel != ".."
+                   && !rel.StartsWith("../")
+                   && !rel.StartsWith(@"..\")
+                   && !Path.IsPathRooted(rel);
+        }
+        
+        /// <summary>
+        /// Returns path ended with "\"
+        /// </summary>
+        /// <param name="path"></param>
+        /// <returns></returns>
+        public static string EnsureTrailingSlash(this string path)
```

Comment:
nice, this will be helpful because through out explorer there are dup code to add trailing slash.